packages/shared/src/loadClerkJsScript.ts

```diff
@@ -36,6 +50,10 @@ const loadClerkJsScript = (opts: LoadClerkJsScriptOptions) => {
     });
   }
 
+  if (!opts?.publishableKey) {
+    throw new Error(MISSING_PUBLISHABLE_KEY_ERROR);
```

Comment:
Why did we switch from errorThrower (prev implementation) to this?

Reply:
Good question! I think I removed it since we'd have to add a different `packageName` for each SDK and didn't look much further on how to 👀 

Looking back at it now, we can set a default `packageName` (`@clerk/clerk-js`?) and override it in the SDK level using `setPackageName`

```
errorThrower.setPackageName('@clerk/astro'`);
```

Reply:
Updated! Please see my comment [here](https://github.com/clerk/javascript/pull/3849#discussion_r1707355166).